### PR TITLE
Feature/gh 689 a4c 3

### LINKED ISSUE #689: Support Alien4Cloud 3.0
The default version of Alien4Cloud installed by Yorc 4.1 bootstrap should now be a 3.0.x Alien4Cloud to benefit from Alien4Cloud new features and bug fixes

### PR DESCRIPTION
**Pull request by laurentganne**
# Pull Request description

### What I did

`versions.yaml`
Updated Alien4Cloud version

`tosca_types.zip`
Updated Alien4Cloud types

### How to verify it

Bottstrap yorc, keeping bootstrap default value for Alien4Cloud URL.
Check the bootrapped setup runs Alien4Cloud 3.0.0

### Description for the changelog

Support Alien4Cloud 3.0 ([GH-689](https://github.com/ystia/yorc/issues/689))

## Applicable Issues

Closes #689

### CODE REVIEW

File: testdata/ci/go.mod
Comment (loicalbertin) [inline testdata/ci/go.mod:8]: maybe we should wait for a milestone before merging this PR

File: testdata/ci/runner/Jenkinsfile
Comment (loicalbertin) [inline testdata/ci/runner/Jenkinsfile:197]: I think the `3` is not necessary as `getURLFromPart` takes the latest one and it may be confusing in the future

File: testdata/ci/go.mod
Comment (laurentganne) [reply testdata/ci/go.mod:8]: Exactly, I'm waiting for a review of this PR: https://github.com/alien4cloud/alien4cloud-go-client/pull/34 :-)
This is why I didn't set you yet as a reviewer here. I just prepare the PR to run continuous integration on it.
I should have tag it as draft in addition

File: testdata/ci/runner/Jenkinsfile
Comment (laurentganne) [reply testdata/ci/runner/Jenkinsfile:197]: ah ok good, I will remove this change

**Review by loicalbertin — APPROVED** [review]
LGTM :+1: :clap:

**Pull request merged**
